src/models/AppState.js

```diff
@@ -261,7 +261,8 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
         deps: ['lastAuthResponse', 'factors'],
         fn: function (res, factors) {
```

Comment:
looks like could depend on `isMfaRequired`, `isMfaChallenge` and `isUnauthenticated`.

Reply:
Thanks for reviewing. I was planning on making that change. I am still making some changes/fixing some tests. Will send out for review once i am done.